packages/testing/src/mocks.ts

```diff
@@ -66,8 +67,11 @@ export const createMock = <T extends object>(
 
   const proxy = new Proxy(partial, {
     get: (obj, prop) => {
+      if (prop === 'constructor') {
+        return () => undefined;
+      }
```

Comment:
This solves our issue. But i'm not sure if it doesn't break anything else.

Is there a better way to solve this?

Reply:
see #587